pkg/oc/cli/admin/admin.go

```diff
@@ -127,9 +126,7 @@ func NewCommandAdmin(name, fullName string, f kcmdutil.Factory, streams genericc
 	// Deprecated commands that are bundled with the binary but not displayed to end users directly
 	deprecatedCommands := []*cobra.Command{
 		// these will be removed soon
-		admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, streams),
 		node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, streams),
```

Comment:
Kill manage node as well (can be separate PR)

Reply:
Yup, that's the plan.

Reply:
This and ipfailover too.